### PR TITLE
Initial Release - Azure RI Purchase Recommendations

### PR DESCRIPTION
**Pull request by dfrankel33**
### Description

Initial Release of Azure RI Purchase Recommendations policy 

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by tusharshahrs** [comment]
ROL tested.  here is the output  https://cl.ly/fb748dc746a7

File: cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  category "RI"
+  label "Net Savings Threshold"
+  description "Specify the minimum net savings that should result in a recommendation"
+  default 100
+  type "number"
+end
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt]: It was decided a while ago that all polices have the label and description.  In this case this the label is the description, and the label should be short "Email addresses to notify".

File: cost/azure/reserved_instances/recommendations/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - `AZURE_EA_KEY` - the Azure EA key for the enrollment being checked
+
+### Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Enrollment ID* - the Azure EA enrollment ID
+- *Look Back Period* - Specify the number of days of past usage to analyze.
+- *Net Savings Threshold* - Specify the minimum Net Savings that should result in an RI purchase recommendation
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/recommendations/README.md]: should match new label as suggested in the .pt file.

File: cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+category "Cost"
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+parameter "param_enrollment" do
+  type "string"
+  label "Azure Enrollment ID"
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt]: Needs description

File: cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+end
+
+parameter "param_days" do 
+  category "RI"
+  label "Look Back Period"
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt]: needs description

**Pull request merged**